project.clj

```diff
@@ -2,6 +2,6 @@
   :description "docopt creates beautiful command-line interfaces - clojure port"
   :url "http://docopt.org"
   :license {:name "MIT" :url "https://github.com/docopt/docopt.clj/blob/master/LICENSE"}
-  :dependencies [[org.clojure/clojure "1.4.0"]]
-  :profiles {:test {:dependencies [[org.clojure/data.json "0.2.1"]]}}
+  :dependencies [[org.clojure/clojure "1.10.0"]]
```

Comment:
```suggestion
  :dependencies [[org.clojure/clojure "1.10.3"]]
```

Reply:
I did it like this to avoid pulling a too new version of Clojure, but thinking about it it shouldn't be a problem.